packages/modules/create-imodel/src/components/base-imodel/BaseIModel.scss

```diff
@@ -62,8 +69,3 @@
   height: 100%;
   justify-content: space-between;
 }
-
-.iac-imodel-input-element {
```

Comment:
dont understand these changes..
Sure you needed to change all the css?
You have left this class in the code...

Reply:
Yeah these are needed, otherwise the elements are not aligned correctly

Reply:
I think I know why its working for you, because there are wrapper props in IModelDescription and IModelTitle components and that class is providing the same styles in CreateIModel.css, try removing that file from the CreateIModel.stores.tsx and everything will be messed up for you as well so we do need wrapper props

Reply:
Just removing this class from CreateIModel.stories.tsx
![image](https://github.com/iTwin/admin-components-react/assets/64415995/259e031a-0a4a-4ea9-b9d6-6f1c30f72762)

Makes the UI like this
![image](https://github.com/iTwin/admin-components-react/assets/64415995/4433f8de-298c-43db-bcb9-b403629cbe33)
![image](https://github.com/iTwin/admin-components-react/assets/64415995/038331fd-1a51-4ebf-a896-0458f7b03ac6)
![image](https://github.com/iTwin/admin-components-react/assets/64415995/035d658e-c20c-4dbd-8421-1d3f93daf847)

Reply:
I copied the contents of BaseIModel.scss from main branch so it should have the same contents as before and it is causing the issues